modules/claims_api/spec/requests/v2/veterans/power_of_attorney/2122_spec.rb

```diff
@@ -553,7 +553,7 @@
                   expect(response).to have_http_status(:unprocessable_entity)
                   expect(response_body['title']).to eq('Unprocessable entity')
                   expect(response_body['status']).to eq('422')
-                  expect(response_body['detail']).to eq(detail)
+                  expect(response_body['detail']).to include(detail)
```

Comment:
Switched to `include` here as the new error message detail includes the position where the error was found (which is what `detail` is here).